bin/webpack.js

```diff
@@ -168,7 +170,23 @@ yargs.parse(process.argv.slice(2), (err, argv, output) => {
 		argv["display"] = "verbose";
 	}
 
-	var options = require("./convert-argv")(yargs, argv);
+	try {
+		Error.stackTraceLimit = 30;
```

Comment:
Not sure if it's a good idea to set this global variable here.

Reply:
@sokra the same thing is happening near webpack compiler call lower in the file: 

https://github.com/webpack/webpack/blob/801a25392d99a8ee7f4371c88d7758169b1ac16f/bin/webpack.js#L331

Maybe I can remove both of them and move it to the top of the file?